### DISCUSSION TITLE
Polls: support multiple choices

### DISCUSSION THREAD

**Discussion by rfc2822** [community/community#52039] [Discussions]
### Select Topic Area

Product Feedback

### Body

Hi,

Suggestion for the Polls (which I like to use in company-internal discussions): if they would allow multiple choices, they could also be used to for instance show availablity of dates for people ("Doodle"). Please support that :)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @rfc2822 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by LindseyB** [comment]
Hi, I've passed this feedback on to the team.

**Reply by Snailedlt** [reply]
Any updates or an ETA? is it in the backlog, or low priority?

**Reply by 100pearlcent** [reply]
Any update on this?

**Reply by rugk** [reply]
This is definitively needed! (It's a shame [my poll now needs to use radio buttons instead of multiple-choice checkboxes…](https://github.com/PrivateBin/PrivateBin/issues/639#issuecomment-2101654415)).

**Reply by Snailedlt** [reply]
> Hi, I've passed this feedback on to the team.

@LindseyB can we at least get a response?

It's been 2 years without any updates at all... And this is not the first time a GitHub staff member doesn't answer questions in the community. Perhaps your company culture needs a reform?

**Reply by LindseyB** [reply]
I don't work on this team anymore

**Reply by Snailedlt** [reply]
@LindseyB thanks for the quick response. Could you perhaps forward it to the right team then?
We've heard no news since your last comment

**Reply by LindseyB** [reply]
They are aware of the feature request since I have worked on it.

**Comment by karasowles** [comment]
+1 I would also like to see this feature!  It's hard for me to make use of the Polls feature without multiple choice checkboxes. Thanks!

**Comment by blanho** [comment]
+1 Makin use of the Polls feature with multiple choice questions is nessecary for me. Hope to hear from u soon

**Comment by bjhartin** [comment]
+1 This would be very useful to us.  We end up using other tools for some polls for this reason.
